client-app/ui-kit/components/atoms/action-dropdown-menu/vc-action-dropdown-menu.vue

```diff
@@ -1,3 +1,4 @@
+<!-- @deprecated the component will be removed from ui-kit -->
```

Comment:
https://github.com/VirtoCommerce/vc-theme-b2b-vue/blob/c31ea77061d7407bf6b1cb385c37c7a8c190061c/client-app/ui-kit/components/atoms/types.d.ts#L7
Put it here instead please